.github/CODEOWNERS

```
@@ -5,13 +5,13 @@
 *                                        @TooTallNate @EndangeredMassa @trek
 /.github/workflows                       @TooTallNate @EndangeredMassa @trek @ijjk
 /packages/fs-detectors                   @TooTallNate @EndangeredMassa @trek @agadzik @chloetedder
-/packages/next                           @TooTallNate @EndangeredMassa @Ethan-Arrowood @trek @ijjk @ztanner
+/packages/next                           @TooTallNate @EndangeredMassa @trek @timneutkens @ijjk @ztanner @huozhi @Ethan-Arrowood
 /packages/routing-utils                  @TooTallNate @EndangeredMassa @trek @ijjk
 /packages/static-build                   @TooTallNate @EndangeredMassa @trek
-/packages/edge                           @vercel/compute @TooTallNate @EndangeredMassa @trek
-/examples                                @leerob
-/examples/create-react-app               @Timer
-/examples/nextjs                         @timneutkens @ijjk @ztanner @huozhi
+/packages/edge                           @TooTallNate @EndangeredMassa @trek @vercel/compute
+/examples                                @TooTallNate @EndangeredMassa @trek @leerob
+/examples/create-react-app               @TooTallNate @EndangeredMassa @trek @Timer
+/examples/nextjs                         @TooTallNate @EndangeredMassa @trek @timneutkens @ijjk @ztanner @huozhi @Ethan-Arrowood
```

```suggestion
/examples/nextjs                         @TooTallNate @EndangeredMassa @trek @timneutkens @ijjk @ztanner @huozhi @Ethan-Arrowood @styfle
```

Adding myself here too

---

Good call: https://github.com/vercel/vercel/pull/11272/commits/bc5c4c6d13ba4143fb984176774ac7bb14f3596d